views/default/admin/users/newest/time_cell.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Render a TD element for an item
+ *
+ * @uses $vars['item']      The item being rendered
+ * @uses $vars['item_vars'] Vars received from the page/components/table view
+ * @uses $vars['attrs']     Array of attributes for the TD
+ */
+
+if (empty($vars['item'])) {
+	// requested as an admin page
+	forward('', '404');
```

Comment:
The columns include the "<td>". That's the one thing I'm not crazy about.

Strawman: if cell rendering doesn't start with `<td` or `<th`, wrap it with a TD.